robot/Cumulus/tests/browser/recurring_donations/zedit_effective_date_erd_record.robot

```diff
@@ -0,0 +1,127 @@
+
+        ${effective_date_initial} =           Get Current Date      result_format=%-m/%-d/%Y
+        ${date} =           Get Current Date
+        ${DATE}=            Add Time To Date      ${date}  30 days      result_format=%Y-%m-%d %H:%M:%S.%f
+        ${date_to_update} =   Convert Date                   ${DATE}                         result_format=%Y-%m-%d
+        ${effective_modified_date}=        Add Time To Date      ${date}  0 days     result_format=%-m/%-d/%Y
+        ${currdate}=        Get Current Date  result_format=datetime
+        ${currentvalue} =   Evaluate          (${currdate.month-1}) * 100
+        ${currentvalue_edited}=   Evaluate    (${currdate.month}*100) + 150
+        Set Suite Variable  ${currentvalue}
+        Set Suite Variable  ${DATE}
+        Set Suite Variable  ${effective_modified_date}
+        Set Suite Variable  ${currentvalue_edited}
+        Set Suite Variable  ${effective_date_initial}
+        Set Suite Variable  ${date_to_update}
+
+        &{contact1_fields}=   Create Dictionary                     Email=rd2tester@example.com
+        &{recurringdonation_fields} =	Create Dictionary           Name=ERDTest1
+        ...                                                         npe03__Installment_Period__c=Monthly
+        ...                                                         npe03__Amount__c=100
+        ...                                                         npe03__Open_Ended_Status__c=${type}
+        ...                                                         Status__c=Active
+        ...                                                         Day_of_Month__c=15
+        ...                                                         InstallmentFrequency__c=${frequency}
+        ...                                                         PaymentMethod__c=${method}
+
+        Setupdata   contact         ${contact1_fields}             recurringdonation_data=${recurringdonation_fields}
+
+
+*** Variables ***
+${frequency}  1
+${day_of_month}  15
+${amount_to_update}  150
+${method}  Credit Card
+${type}    Open
+
+*** Test Cases ***
+
+Edit An Enhanced Recurring donation record of type open
+    [Documentation]               After creating an open recurring donation using API, The test ensures that when the record
+     ...                          is edited and effective date is updated to next month , Verifies that future schedule is shown
+     ...                          with the right values reflected. Verifies the current year and future year schedules are updated
+     ...                          Verifies the opportunity status
+
+
+    [tags]                                   unstable               W-041167            feature:RD2
+
+    Go To Page                               Details
+    ...                                      npe03__Recurring_Donation__c
+    ...                                      object_id=${data}[contact_rd][Id]
+    Wait Until Loading Is Complete
```

Comment:
This keyword is already included in the implementation of "Go to page" for Detail page in CumulusCI framework. So you can remove this from your test. Can you remove all occurrences of this from the file ?

Reply:
removed